test/integration/helpers/PathToNode

```diff
@@ -0,0 +1 @@
+appium/index.js
```

Comment:
Not sure about the file name here. This is the path to appium package index, not to the node

Reply:
PathToPackageIndex?